cluster-autoscaler/debuggingsnapshot/README.md

```diff
@@ -0,0 +1,46 @@
+# Debugging Snapshotter
+ It's a tool to visualize the internal state of cluster-autoscaler at a point in time to help debug autoscaling issues.
+
+---
+### Requirements
+Require Cluster-autoscaler versions 1.24+
+
+---
+#### What data snapshotter can capture?
+https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/debuggingsnapshot/debugging_snapshot.go#L60C1-L71C1
+```go
+type DebuggingSnapshotImpl struct {
+	NodeList                      []*ClusterNode          `json:"NodeList"`
+	UnscheduledPodsCanBeScheduled []*v1.Pod               `json:"UnscheduledPodsCanBeScheduled"`
+	Error                         string                  `json:"Error,omitempty"`
+	StartTimestamp                time.Time               `json:"StartTimestamp"`
+	EndTimestamp                  time.Time               `json:"EndTimestamp"`
+	TemplateNodes                 map[string]*ClusterNode `json:"TemplateNodes"`
+}
+
+```
+
+#### Development
+1. First step is enable the snapshotter adding the following flag to cluster-autoscaler [manifest](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/bizflycloud/manifest/cluster-autoscaler.yaml) .
```

Comment:
We should avoid referring to a single cloud provider implementation's of the manifest.
```suggestion
1. First step is to enable the snapshotter feature by adding the following flag to cluster-autoscaler.
```